runbot_docker_orchest/controllers/runbot_hook.py

```diff
@@ -0,0 +1,24 @@
+from odoo.http import request
+from odoo import http
+import logging
+import json
+
+_logger = logging.getLogger(__name__)
+
+
+class RunbotQuayioHook(http.Controller):
+
+    @http.route('/runbot/quayio_hook',  type='json', auth='public')
+    def quayio_hook(self):
+        hook_data = request.jsonrequest
+        _logger.info(
+            'Trigger webhook request json %s', json.dumps(hook_data, indent=2))
+        repo_name = hook_data.get('repository')
+        branches = hook_data.get('updated_tags')
+        branches = (
+            isinstance(branches, dict) and list(branches.keys()) or branches)
+        repo = request.env['runbot.repo'].sudo().search(
+            [('quayio_repo_name', '=', repo_name)])
+        branch = request.env['runbot.branch'].search(
+            [('name', 'in', branches), ('repo_id', '=', repo.id)])
+        branch.sudo().write({'updatable': True})
```

Comment:
Use the same logic from:
https://github.com/odoo/runbot/blob/4a558741e01b206152ed10f5062d0fc94121fed5/runbot/controllers/hook.py#L15

I mean, write the current date in the `repo.hook_time` field
Remove `updatable` field.